packages/frama-c/frama-c.20150201/descr

```diff
@@ -7,3 +7,5 @@ collaborative framework. The collaborative approach of Frama-C allows
 static analyzers to build upon the results already computed by other
 analyzers in the framework. Thanks to this approach, Frama-C provides
 sophisticated tools, such as a slicer and dependency analysis.
+
+This virtual package ensures the ide to be available.
```

Comment:
Proposed change of wording:
`This package forces the installation of frama-C with its IDE.`